EnergyPlus_oM/LocationAndClimate/SiteLocation.cs

```diff
@@ -26,9 +26,9 @@
 using BH.oM.Reflection;
 
 namespace BH.oM.Adapters.EnergyPlus
-{
-    public class SiteLocation : BHoMObject, IEnergyPlusClass
-    {
+{    public class SiteLocation : BHoMObject, IEnergyPlusClass
+
```

Comment:
Remove newline from here.

Reply:
The bracket `{` and class definition `public class SiteLocation` should be on different lines